src/components/common/Header/Header.styles.tsx

```diff
@@ -0,0 +1,40 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+export const Root = styled.header`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 80px;
+
+  background-color: black;
+`;
+
+export const LinkToHome = styled(Link)`
+  display: flex;
+  align-items: center;
+`;
```

Comment:
Flex 를 만드셨는데, 스타일을 상속하실 수도 있습니다! (꼭 바꾸라는건 아닙니당!)

```suggestion
export const Root = styled(Flex)`
  width: 100%;
  height: 80px;

  background-color: black;
`;

export const LinkToHome = styled(Flex)``;
```

Reply:
실제로 `Flex`를 상속받아 사용하는 방법도 생각해 보았는데요!
다만 `Flex`에서 사용되는 속성 이상의 속성들이 필요한 경우 `StyledComponent`를 사용함에도 인라인 속성도 함께 늘어나 가독성이 떨어지지 않을까..하는 고민을 하다가 적용하지 않았던 것 같습니다!

```jsx
return (
  <S.Root justify="center" align="center" width="100%" height="80px" ...>
     ...
  </S.Root>
)
```
혹시 헤인티는 이런 경우 어떤 방식으로 해결하는걸 선호하실까요~?!

Reply:
아~! 저는 아래와 같이 사용하기도 합니다!

```javascript
export const Root = styled(() => <Flex justify="center" align="center" />)`
  width: 100%;
  height: 80px;
  background-color: black;
`;
```